cypress/e2e/ArticleQuiz/GuestEarnCredits.cy.js

```diff
@@ -1,6 +1,6 @@
 import { faker } from '@faker-js/faker';
 
-describe('GuestEarnCredits', () => {
+describe('GuestEarnCredits', { tags: ['@article-quiz', '@guest', '@business:medium-impact'] }, () => {
```

Comment:
Not sure about this one, because they could be potential users, but not users at all.